src/amo/components/Addon/index.js

```diff
@@ -500,7 +500,7 @@ export class AddonBase extends React.Component {
               />
             ) : null}
             {addon && addon.status !== STATUS_PUBLIC ? (
-              <Notice type="error">
+              <Notice type="error" className="Non-public-addon-notice">
```

Comment:
I think it should start with `Addon`, like `Addon-non-public-notice`.

Reply:
:man_facepalming: Done :)